README.md

```diff
@@ -3743,10 +3775,10 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
-## ECMAScript 5 Compatibility
+## ECMAScript 6 Compatibility
 
-  <a name="es5-compat--kangax"></a><a name="26.1"></a>
-  - [27.1](#es5-compat--kangax) Refer to [Kangax](https://twitter.com/kangax/)’s ES5 [compatibility table](https://kangax.github.io/es5-compat-table/).
```

Comment:
existing links can't ever change, so please revert that part.

Reply:
Which link do you mean? Link to the https://twitter.com/kangax/ ?

We have already replaced 
_https://kangax.github.io/es5-compat-table_
with 
_https://compat-table.github.io/compat-table/es5/_

Do we still need to mention here Kangax?

Reply:
The anchor link - `es5-compat--kangax` has to work for eternity (as should all URLs)